app/services/courses_query.rb

```diff
@@ -65,6 +67,29 @@ def study_modes_scope
     end
   end
 
+  def qualifications_scope
+    return @scope if params[:qualifications].blank?
+
+    @applied_scopes[:qualifications_scope] = params[:qualifications]
+
+    case params[:qualifications]
+    when ['qts']
+      @scope.where(qualification: [Course.qualifications[:qts]])
+    when ['qts_with_pgce_or_pgde'], ['qts_with_pgce']
+      @scope.where(qualification: [Course.qualifications[:pgce_with_qts], Course.qualifications[:pgde_with_qts]])
```

Comment:
`qts_with_pgce` should also match `pgde_with_qts`?

Reply:
Yes, 😞 

This is the live mechanism of the current search. (See commit message with the whole explanation). Happy to huddle to explain in case the commit message is not clear.

Reply:
All good. Thanks!